inductiva/_cli/cmd_tasks/list.py

```diff
@@ -19,15 +21,37 @@ def list_tasks(args):
 
     table_dict = tasks.to_dict(task_list)
 
+    emph_formatter = inductiva.get_ansi_formatter()
+
+    def color_formater(status):
+        if status == models.TaskStatusCode.SUCCESS:
+            return emph_formatter(status, utils.format_utils.Emphasis.GREEN)
+        elif status in [
+                models.TaskStatusCode.FAILED, models.TaskStatusCode.KILLED,
+                models.TaskStatusCode.EXECUTERFAILED,
+                models.TaskStatusCode.EXECUTERTERMINATED,
+                models.TaskStatusCode.EXECUTERTERMINATEDBYUSER,
+                models.TaskStatusCode.SPOTINSTANCEPREEMPTED,
+                models.TaskStatusCode.ZOMBIE
+        ]:
+            return emph_formatter(status, utils.format_utils.Emphasis.RED)
+        return status
+
     formatters = {
-        "Submitted": utils.format_utils.datetime_formatter,
-        "Started": utils.format_utils.datetime_formatter
+        "Submitted": [utils.format_utils.datetime_formatter,],
+        "Started": [utils.format_utils.datetime_formatter,],
+        "Status": [color_formater]
     }
 
-    print(utils.format_utils.get_tabular_str(
-        table_dict,
-        formatters=formatters,
-    ))
+    header_formatters = [
+        lambda x: x.upper(),
+        lambda x: emph_formatter(x, utils.format_utils.Emphasis.BOLD)
+    ]
```

Comment:
you can do both things in the same call to a single formatter:

```suggestion
    header_formatters = [
        lambda x: emph_formatter(x.upper(), utils.format_utils.Emphasis.BOLD)
    ]
```